torch/functional.py

```diff
@@ -596,7 +600,7 @@ def stft(input: Tensor, n_fft: int, hop_length: Optional[int] = None,
             frames. Default: ``None`` (treated as equal to ``floor(n_fft / 4)``)
         win_length (int, optional): the size of window frame and STFT filter.
             Default: ``None``  (treated as equal to :attr:`n_fft`)
-        window (Tensor, optional): the optional window function.
+        window (Tensor, str, optional): the optional window function.
```

Comment:
This is misleading (it was misleading before, too) because this PR will allow users to specify a window function to generate the window OR to provide a tensor to be used as the window directly